app/component/map/tile-layer/bikeParks.js

```diff
@@ -2,11 +2,11 @@ import { graphql } from 'react-relay';
 
 export default graphql`
   query bikeParksQuery($ids: [String!]!) {
```

Comment:
Is this used somewhere? I tried to do a search in the code base for bikeparks but I found nothing.

Reply:
Should I remove this as unused code? Indeed, there seems to be not a single reference to query.